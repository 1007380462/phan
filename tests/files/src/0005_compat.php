<?php

class A {

    /**
     * @param $collection
     */
    public function f($a = []) {
    }

    /**
     * @param A $a
     * @param int $b
     * @param string $c
     */
    public function g(A $a, $b = false, $c = -1) {
    }
}
