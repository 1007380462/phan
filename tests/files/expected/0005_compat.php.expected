%s:16 TypeError Default value for int $b can't be bool
%s:16 TypeError Default value for string $c can't be int
%s:26 ParamError required arg follows optional
